docker/puppetserver-standalone/docker-entrypoint.d/90-consul.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+  CONSUL_HOSTNAME="${CONSUL_HOSTNAME:-consul}"
+  CONSUL_PORT="${CONSUL_PORT:-8500}"
+
+  cat <<SERVICEDEF > /puppet-service.json
+{
+  "name": "puppet",
+  "id": "$HOSTNAME",
+  "port": 8140,
+  "address": "$ipaddress",
+  "checks": [
+    {
+      "http": "https://${HOSTNAME}:8140/${PUPPET_HEALTHCHECK_ENVIRONMENT}/status/test",
```

Comment:
The healthcheck that hits this same endpoint includes the various SSL files to satisfy HTTPS.

Is that not necessary here with Consul? Is that what `tls_skip_verify` is for?

Reply:
yeah, `tls_skip_verify=true` is basically like passing `--insecure` to curl like we do in the pdb entrypoint.